### ISSUE TITLE
inconsistent usage between specifications and example codes

### ISSUE THREAD

**Issue by LinLabNEU** [CoCoPIE-Group/XGen-Report#29]
In the manual with the link https://xgen.cocopie.ai/v1.1.0/4_Usage/#step-iii-prepare-training-scripts , in the part of Step III: Prepare Training Scripts in custom AI, in step 3, it claims that we should use the following,
```
originalArgs, args_ai = xgen_init(user_args, args_ai, COCOPIE_MAP)
```
But in the following example code, it becomes
```
user_args,args_ai = xgen_init(orginalArgs,map = COCOPIE_MAP)
```
They are inconsistent. 

Besides, it has a new variable `user_args`. But in the optimizer, it still uses `lr=args.learning_rate` where `args` is not `user_args` or `orginalArgs`.

**Comment by Ucag** [comment]
The signature of `xgen_init` is `xgen_init(user_args, args_ai=None, map = None)->Union[dict,Namespace]`, the example ignored optional parameter. 

As for `lr=args.learning_rate`, if want XGen user to control the learning rate, `lr` should be in the `COCOPIE_MAP`. Then use it like `lr=args_ai.lr`. The example only mapped `{'train_epochs' : 'common_train_epochs'}` in `COCOPIE_MAP`.

Thanks for your report, the doc is updated due to its confusion to the reader. API usage is written a more clearer way.
